fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/JDBCQueryBuilder.java

```diff
@@ -248,12 +248,22 @@ protected JDBCOperator getOperator(Parameter queryParm) {
         JDBCOperator operator = null;
         Modifier modifier = queryParm.getModifier();
 
-        if (modifier != null) {
+        // In the case where a URI, we need specific behavior/manipulation
+        // so that URI defaults to EQ, unless... BELOW
+        if (Type.URI.compareTo(queryParm.getType()) == 0) {
+            if(modifier != null && Modifier.BELOW.compareTo(modifier) == 0) {
```

Comment:
according to the spec https://www.hl7.org/fhir/search.html, we should consider both "below" and "above":
"For uri: :below, :above indicate that instead of an exact match, either the search term left-matches the value, or vice-versa."
I guess "left-matches" means begin with, and "right-matches" means end-with ...

Reply:
Above would be a totally different implementation, and require some re-coding

I think we can add the support doing the following: 

- If Modifier is above, switch to LIKE
- Find the domain of the URL using `/` 
     http://acme.org/fhir/ValueSet/123/_history/5
- Use a BIND value of `http://acme.org/`

Does this approach make sense?

Reply:
I'm not sure what's the real meaning of left-match and right-match here... @JohnTimm , what's your suggestion?

Reply:
BTW, seems the null check is not needed here...

Reply:
It's not a left-right match with above. 
It's matching on the individual components

Reply:
PR https://github.com/IBM/FHIR/pull/new/issue-448